components/mpas-seaice/src/shared/mpas_seaice_advection_incremental_remap.F

```diff
@@ -4663,6 +4663,9 @@ subroutine compute_barycenter_coordinates(&
        mean1, center1, xGrad1, yGrad1,  &
        mean2, center2, xGrad2, yGrad2)
 
+    use seaice_constants, only: &
+         seaicePuny
```

Comment:
Curious, seaicePuny seems like an existing constant? Just wondering what value was chosen and any rationale.

Reply:
@sarats - it was just an existing constant that at least seemed appropriate. I'm open to other ideas -- is there a reasonable constant for which the inverse will always be OK for all compilers?